notebooks/en/rag_evaluation.ipynb

```diff
@@ -37,47 +37,76 @@
   },
```

Comment:
<p>Here, you can also remove the output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>